testIntegration/package.json

```diff
@@ -15,7 +15,9 @@
     "bundle": "rm -rf commerce-sdk*.tgz; npm pack .. && mv commerce-sdk-*.tgz commerce-sdk.tgz",
     "bundle:install": "npm run bundle && npm install",
     "bundle:clean:install": "npm run clean:all && npm run bundle:install",
-    "test:bundle": "npm run bundle:install && npm run test",
+    "test:prod": "npm run bundle:install && npm run test",
+    "test:staging": "COMMERCE_SDK_INPUT_DIR=testIntegration/stagingApis npm run test:prod",
```

Comment:
Note that `testIntegration/stagingApis` is correct, due to a limitation of gulp.